src/include/processor/operator/base_shortest_path.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "queue"
+
+#include "processor/operator/physical_operator.h"
+#include "storage/storage_structure/lists/lists.h"
+
+namespace kuzu {
+namespace processor {
+
+struct BFSState {
```

Comment:
Set initialized values for these variables.

Reply:
added constructor to this struct for initializing the variables